core/common-ui/src/main/java/com/tgyuu/common/util/PlayStoreUtil.kt

```
@@ -0,0 +1,25 @@
+import androidx.core.net.toUri
+
+fun Context.openPlayStore() {
+    try {
+        val playStoreIntent = Intent(
+            Intent.ACTION_VIEW,
+            "market://details?id=$packageName".toUri()
+        ).apply {
+            addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
+        }
+        startActivity(playStoreIntent)
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)